.github/workflows/ci.yml

```diff
@@ -106,16 +106,10 @@ jobs:
     strategy:
       matrix:
         include:
-          - python-version: [ 3, 11 ]
-            tox-env: py311-pip20
-            tox-env-python: python
-          - python-version: [ 3, 11 ]
-            tox-env: py311-pip20-integration
-            tox-env-python: python
-          - python-version: [ 3, 12, "0-rc.3" ]
+          - python-version: [ 3, 12 ]
             tox-env: py312-pip23_2
             tox-env-python: python3.11
```

Comment:
Just confirming that these are intentionally remaining at `python3.11`?

Reply:
Yup. The removal of distutils from the stdlib in Python 3.12 is fairly disruptive to build tools. This allows tox to work.

Reply:
For reference: https://github.com/pantsbuild/pex/pull/2172/files#r1254738095